.github/workflows/build-plugin-zip.yml

```
@@ -77,7 +77,7 @@ jobs:
               id: get_version
               run: |
                   OLD_VERSION=$(jq --raw-output '.version' package.json)
-                  echo "::set-output name=old_version::$(echo $OLD_VERSION)"
+                  echo "old_version=$(echo $OLD_VERSION)" >> $GITHUB_OUTPUT
```

Double echo is likely not needed.

```suggestion
                  echo "old_version=${OLD_VERSION}" >> $GITHUB_OUTPUT
```

Verify to be sure.

---

This seems to be OK.